packages/web-react/src/components/Tooltip/README.md

```diff
@@ -0,0 +1,231 @@
+# Tooltip
+
+## Usage
+
+### Basic
+
+```javascript
+<TooltipWrapper>
+  <button type="button" className="Button Button--primary Button--medium TooltipTarget">
+    I have a tooltip!
+  </button>
+  <Tooltip>Hello there!</Tooltip>
+</TooltipWrapper>
+```
+
+### Dismissible
+
+```javascript
+import { useState } from 'react';
+
+const [open, setOpen] = useState(true);
+const closeHandler = () => setOpen(false);
+
+<TooltipWrapper>
+  <button type="button" className="Button Button--primary Button--medium">
+    I have a tooltip!
+  </button>
+  <Tooltip isDismissible open={open} onClose={closeHandler}>
+    Hello there!
+  </Tooltip>
+</TooltipWrapper>;
+```
+
+### Clickable
```

Comment:
I am not sure this is the correct title. I would expect `Clickable` means that you can click on something inside the tooltip.

Reply:
fixed